src/axom/sina/docs/sphinx/documents.rst

```diff
@@ -106,6 +106,30 @@ of the ``Document`` that way:
         std::cout << myDocument.toJson() << std::endl;
     }
 
+
+------------------------------
+Generating Documents From HDF5
+------------------------------
+
+In addition to assembling ``Document`` instances from existing JSON files, it
+is possible to generate ``Document`` objects from existing HDF5 files using
+conduit.
+
+Sina's ``saveDocument()`` and ``loadDocument()`` functions support HDF5 assembly if we both 
+build axom with HDF5 support and provide it with the optional Protocol variable set to HDF5. 
+If Protocol::HDF5 is attempted while axom does not have HDF5 support, the functions will 
+recognize they won't have the support for what you are attempting and will return a list of 
+supported types instead as a runtime error.
```

Comment:
Minor: It might be clearer to discuss an Axom _configuration_ with HDF5. I.e. Axom supports HDF5, but could be configures with or without HDF5 enabled.

Something like the following might be clearer:
```suggestion
When Axom is configured with HDF5 support, Sina's ``saveDocument()`` and ``loadDocument()`` functions support HDF5 assembly through the `Protocol::HDF5` argument.  The functions will throw a runtime error with the list of available types in Axom configurations without HDF5 support.
```